ddtrace/_trace/trace_handlers.py

```diff
@@ -769,6 +812,14 @@ def listen():
     core.on("rq.queue.enqueue_job", _propagate_context)
 
     for context_name in (
+        # web frameworks
+        "aiohttp.request",
+        "bottle.request",
+        "cherrypy.request",
+        "falcon.request",
+        "molten.request",
+        "pyramid.request",
+        "sanic.request",
```

Comment:
Would it be possible to refactor the django and flask integrations to use the new trace_handlers. It would be nice if our trace_handlers were integration agnostic and easily reusable.

Reply:
Definitely possible but out of scope for this PR since both already use core. We would need to significantly refactor their trace handlers since they have a bunch of special logic respectively.